### PR TITLE
mesh_com: Add iperf unit test in mesh network

### PR DESCRIPTION
**Pull request by anshul-tii**

### CODE REVIEW

File: common/tests/mesh_com_1.0/neighbour_count.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        aux = x.split()
+        i += 1
+        if i > 2:
+            macs.append((aux[1]).decode("utf-8"))
+    #print("Neighbour MAC Address list :", macs)
+    count= len(macs)
+    print(len(macs))
+if __name__ == '__main__':
+    get_macs()
+
```
Comment (martin-tii) [inline common/tests/mesh_com_1.0/neighbour_count.py:16]: maybe we can create a 'util.py' file and then import some of these.
@govindsi

File: common/tests/mesh_com_1.0/test_mesh_iperf.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+#######################################
+_init() {
+  test_mesh_com_1.0_dependencies
+  _deinit
+
+  count=`python neighbour_count.py`
+  if [ $count -gt 0 ]
+  then
+     echo "Neigbours Exist and there count is : $count"
+  else
```
Comment (martin-tii) [inline common/tests/mesh_com_1.0/test_mesh_iperf.sh:23]: try to use a unified pattern,  (words with capital or not )

File: common/tests/mesh_com_1.0/test_mesh_iperf.sh [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  esac
+done
+
+echo "Please select protocol"
+PS3="Select among UDP and TCP packet: "
+select opt in UDP  TCP  quit; do
+
+  case $opt in
+    UDP)
+      echo "Generating Stats for UDP Case-----------PLease Wait................................."
```
Comment (martin-tii) [inline common/tests/mesh_com_1.0/test_mesh_iperf.sh:76]: typo in Please

**Review by martin-tii — APPROVED** [review]

File: common/tests/mesh_com_1.0/neighbour_count.py (hunk 1 above)
Comment (govindsi) [reply common/tests/mesh_com_1.0/neighbour_count.py:16]: yes, agreed!

File: common/tests/mesh_com_1.0/test_mesh_iperf.sh (hunk 2 above)
Comment (anshul-tii) [reply common/tests/mesh_com_1.0/test_mesh_iperf.sh:23]: @martin-tii Issues resolved

File: common/tests/mesh_com_1.0/test_mesh_iperf.sh (hunk 3 above)
Comment (anshul-tii) [reply common/tests/mesh_com_1.0/test_mesh_iperf.sh]: @martin-tii Corrected the typo

**Review by govindsi — APPROVED** [review]

**Pull request merged**